src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1533,12 +1526,6 @@ internal override void Check(IHostEnvironment env)
             };
 
             _outputColumns = outCols.ToArray();
-
-        }
-
-        public SdcaBinaryTrainer(IHostEnvironment env, Arguments args)
```

Comment:
>  public SdcaBinaryTrainer(IHostEnvironment env, Arguments args) [](start = 6, length = 64)

i think this is needed for the SignatureTrainer. Doesn't have to be public.

Reply:
the constructor above has this signature. so its not really going away.  

---
In reply to: [248771919](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248771919) [](ancestors = 248771919)